app/templates/app/styles/pages/_styleguide.scss

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   WEBSITE STYLE GUIDE
+   ========================================================================== */
+
+// Global
+.styleguide {
+	#main {
+		@include outer-container;
+		margin: 4em auto;
+	}
+}
+
+// Component
+.component {
+	background-color: $base-background-color;
+  border: 1px solid $base-border-color;
+  margin-bottom: 1em;
+  padding: 1em;
+}
+
+.component__header {
+  h3 {
+    margin-top: 0 !important;
+  }
+
+  p {
+  	font-size: 0.8rem;
+    margin-top: 0;
+    
+    a {
+      border-bottom-width: 0;
+    }
+
+    &.entry--last {
+      margin-bottom: 1em;
+    }
+  }
+}
+
+.component__rendered {
+  &--color {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+
+    @include breakpoint($medium-screen) {
+      flex-wrap: nowrap;
+    }
+  }
+}
+
+.component__code {
+  clear: both;
+  padding-top: 3em;
+  position: relative;
+
+  &:before {
```

Comment:
Unless you plan to support IE8, it's usually clearer to use the [double colon syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements#Notes) for pseudo elements.